exe/atlasq

```diff
@@ -5,6 +5,9 @@ debug = true if ARGV.delete("-d")
 debug = true if ARGV.delete("--debug")
 ENV["ATLASQ_DEBUG"] = "1" if debug
 
+# Avoid warnings on Ruby 2.7 related to pattern matching.
+Warning[:experimental] = false
```

Comment:
It was a real pain tracking this down. This only showed up in integration tests but then it took me a bit to realize that other methods of silencing this didn't work.